src/skins/vector/css/matrix-react-sdk/structures/MatrixChat.css

```diff
@@ -94,7 +94,7 @@ limitations under the License.
     background-color: #eaf5f0;
 
     -webkit-flex: 0 0 210px;
-    flex: 0 0 210px;
+    flex: 0 0 235px;
```

Comment:
I don't think that the `-webkit-flex` is used anymore, but I've corrected anyway